libraries/DataFlash/DataFlash.cpp

```diff
@@ -50,6 +50,12 @@ const AP_Param::GroupInfo DataFlash_Class::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_FILE_DSRMROT",  4, DataFlash_Class, _params.file_disarm_rot,       0),
 
+    // @Param: _MAV_BUFSIZE
+    // @DisplayName: Maximum DataFlash MAVLink Backend buffer size (in kilobytes)
+    // @Description: Maximum amount of memory to allocate to DataFlash-over-mavlink.
+    // @User: Standard
```

Comment:
This should be an advanced parameter. It's not something most users will ever have to alter.

Reply:
Done